### ISSUE TITLE
Incorrect xref to Oncotree for `mondo:0000921` (Ampulla of Vater Neoplasm)

### ISSUE THREAD

**Issue by jsstevenson** [monarch-initiative/mondo#8026]
**Mondo term (ID and Label):**

ampulla of vater neoplasm [http://purl.obolibrary.org/obo/MONDO_0000921]

**Xref that should be fixed (ID and label):**

`ONCOTREE:AMPULLAOFVATER` (should be `ONCOTREE:AMPULLA_OF_VATER`).

The OncoTree API is a little tricky so I'm not sure of a better way to provide evidence for this -- but if you ctrl+f for code "AMPULLA_OF_VATER" [here](https://oncotree.mskcc.org/api/tumorTypes) you should be able to find it

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0002-2568-6163

**Other comments:**

There are a handful of other OncoTree mappings that may have similar issues (ie, missing underscores in xref to oncotree tissue type code) -- `mondo:0005304` (biliary tract neoplasm), `mondo:0021227` (adrenal gland neoplasm), `mondo:0006424`.

**Comment by nicolevasilevsky** [comment]
@twhetzel should we review these and do a more systematic fix?

**Comment by twhetzel** [comment]
@nicolevasilevsky  Yes, we can look at this systematically. If I gather the data, can you review and make the updates?

**Comment by nicolevasilevsky** [comment]
@twhetzel sorry for the delayed reply! Yes, I am happy to review and do the updates, thanks!

**Comment by twhetzel** [comment]
No worries, I will get this into the queue.
